build.gradle

```
@@ -147,6 +149,25 @@ pitest {
     mutationThreshold = 50
 }
 
+checkstyle {
+    maxWarnings = 0
+    toolVersion = '8.21'
```

Probably worth going to the latest? (you'll probably have to make minor changes to your checkstyle.xml
```suggestion
    toolVersion = '8.29'
```